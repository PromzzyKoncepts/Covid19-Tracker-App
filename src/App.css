.cards {
  display: grid;
  grid-template-columns: repeat(5, 1fr);
  margin-left: 20px;
  margin-right: 15px;
  margin-top: 50px;
  padding-bottom: 50px;
  gap: 10px;
  text-align: end;
  align-items: center;
}

.item {
  background-color: #4369B2;
  border-radius: 5px;
  padding: 5px;
  margin: 0;
  width: 250px;
  height: 210px;
}

.flag {
  width: 110px;
  height: 70px;
}

.popupFlag {
  width: 150px;
  height: 100px;
}

.flagBtn {
  display: flex;
  justify-content: space-around;
  align-items: center;
}

.viewBtn {
  width: 20px;
  filter: invert(1);
}

.province {
  font-weight: 700;
  color: #fff;
  margin-right: 15px;
  margin-top: 15px;
}

.country {
  font-weight: 500;
  color: #fff;
  margin-right: 15px;
}

.back {
  width: 30px;
  float: right;
  transform: translate(-60px, -20px);
}

.back:hover {
  filter: invert(1);
  transition: all 0.5s ease-in;
}

.popup-container {
  /* background-color: #1d1d1d; */
  background-image: url(./assets/slide1.jpg);
  height: 100vh;
}

.popup {
  transform: translateY(15%);
  background-color: #436ab2e3;
  width: 45%;
  padding-top: 70px;
  padding-bottom: 30px;
  padding-bottom: 30px;
  text-align: center;
  margin: auto;
  color: #fff;
  border-radius: 8px;
  box-shadow: 0 0 10px #1d1d1d;
}

.region {
  width: 70%;
  margin: auto;
  padding: 20px;
}

.info:hover {
  background: #1d1d1d;
  padding: 6px 0;
  transition: all 0.3s ease-in;
}

.title {
  margin-bottom: 22px;
  margin-top: 10px;
}