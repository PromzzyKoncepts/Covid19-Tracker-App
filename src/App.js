import "./App.css";
import { Routes, Route } from "react-router-dom";
import Home from "./components/Home";

function App() {
  return (
    <Routes>
      <Route path="/" exact element={<Home />} />
    </Routes>
  );
}

export default App;
