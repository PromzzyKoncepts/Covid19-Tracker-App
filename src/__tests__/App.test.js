import React from "react";
import renderer from "react-test-renderer";
import { MemoryRouter as Router } from "react-router-dom";
import App from "../App";
import { Provider } from "react-redux";
import store from "../redux/Store";


it("renders Component correctly", () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Router>
        <App />
      </Router>
    </Provider>
  );
  expect(tree).toMatchSnapshot();
});
