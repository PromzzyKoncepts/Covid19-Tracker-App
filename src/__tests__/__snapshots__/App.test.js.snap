// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Component correctly 1`] = `
<div
  className="wrapper"
>
  <div>
    <div
      className="carousel slide carousel-fade"
      onKeyDown={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
    >
      <div
        className="carousel-indicators"
      >
        <button
          aria-current={true}
          aria-label="Slide 1"
          className="active"
          data-bs-target=""
          onClick={[Function]}
          type="button"
        />
        <button
          aria-current={false}
          aria-label="Slide 2"
          data-bs-target=""
          onClick={[Function]}
          type="button"
        />
        <button
          aria-current={false}
          aria-label="Slide 3"
          data-bs-target=""
          onClick={[Function]}
          type="button"
        />
      </div>
      <div
        className="carousel-inner"
      >
        <div
          className="active carousel-item"
        >
          <img
            alt="Market ratings"
            className="d-block w-100"
            src="slide1.jpg"
          />
          <div
            className="carousel-caption"
          >
            <h3
              className="head"
              style={
                Object {
                  "color": "#fff",
                  "fontSize": "4rem",
                }
              }
            >
              Corona Virus
            </h3>
            <p
              className="text"
              style={
                Object {
                  "color": "#fff",
                  "fontSize": "4rem",
                }
              }
            >
              The Worlds Most Adverse Pandemic
            </p>
          </div>
        </div>
        <div
          className="carousel-item"
        >
          <img
            alt="Ratings"
            className="d-block w-100"
            src="slide2.jpg"
          />
          <div
            className="carousel-caption"
          >
            <h3
              className="head"
              style={
                Object {
                  "background": "#1d1d1d",
                  "color": "#fff",
                  "fontSize": "3rem",
                  "margin": "0px",
                }
              }
            >
              2019-2021
            </h3>
            <p
              className="text"
              style={
                Object {
                  "background": "#4369B2",
                  "color": "#fff",
                  "fontSize": "2rem",
                }
              }
            >
              View Countries with most reported cases
            </p>
          </div>
        </div>
        <div
          className="carousel-item"
        >
          <img
            alt="coin ratings"
            className="d-block w-100"
            src="https://covid19.mathdro.id/api/og"
          />
          <div
            className="carousel-caption"
          >
            <h3
              className="head"
              style={
                Object {
                  "background": "#1d1d1d",
                  "color": "#fff",
                  "fontSize": "3rem",
                  "margin": "0px",
                }
              }
            >
              Covid19 Global Statistics
            </h3>
          </div>
        </div>
      </div>
      <a
        className="carousel-control-prev"
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          aria-hidden="true"
          className="carousel-control-prev-icon"
        />
        <span
          className="visually-hidden"
        >
          Previous
        </span>
      </a>
      <a
        className="carousel-control-next"
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          aria-hidden="true"
          className="carousel-control-next-icon"
        />
        <span
          className="visually-hidden"
        >
          Next
        </span>
      </a>
    </div>
  </div>
  <div
    className="search"
  >
    <div
      className="center input-group input-group-lg"
      onChange={[Function]}
    >
      <span
        className="input-group-text"
        id="inputGroup-sizing-lg"
      >
        Search
      </span>
      <input
        aria-describedby="inputGroup-sizing-sm"
        aria-label="Large"
        className="form-control"
        placeholder="Country name"
      />
    </div>
  </div>
  <div
    className="cards"
  />
  <div
    className="footer"
  >
    <h3
      className="footer-heading"
    >
      Covid19
       
      <span
        className="span"
      >
        Tracker
      </span>
    </h3>
    <p
      className="footer-text"
    >
      All Rights Reserved,
    </p>
    <p
      className="footer-text"
    >
       Â© Promise Okechukwu
    </p>
  </div>
</div>
`;
